expo/tests/gbe/test_edit_combo.py

```diff
@@ -16,7 +16,7 @@ class TestCreateCombo(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
         self.client = Client()
-        self.troupe_string = 'a one-off group of performers working together'
+        self.troupe_string = 'a one-off group of performers working     together'
```

Comment:
I can't replicate your error.  I did the following:
- go to the GBE-463 branch
- sync with github, just in case
- remove all *.pyc files in expo/tests/gbe
- remove all *.pyc files in expo/factories/gbe
- remove all *.pyc files in expo/

And I can't seem to find a way to make the test fail.  I'm willing to dump my vagrant box and try refreshing... but not tonight.